.github/ISSUE_TEMPLATE/unmaintained.yaml

```
@@ -0,0 +1,290 @@
+name: Unmaintained component
+description: Create an issue to track the deprecation of a component
+labels: ["help wanted", "unmaintained"]
+body:
+  - type: markdown
+    attributes:
+      value: |
+        Thanks for taking the time to fill out this bug report! Please make sure to fill out the entire form below, providing as much context as you can in order to help us triage and track down your bug as quickly as possible.
+
+        Before filing a bug, please be sure you have searched through [existing bugs](https://github.com/open-telemetry/opentelemetry-collector-contrib/issues?q=is%3Aissue+is%3Aopen+sort%3Aupdated-desc+label%3Abug) to see if an existing issue covers your bug.
```

```suggestion
        Before filing a bug, please search through [existing bugs](https://github.com/open-telemetry/opentelemetry-collector-contrib/issues?q=is%3Aissue+is%3Aopen+sort%3Aupdated-desc+label%3Abug) to see if an existing issue covers your bug.
```
Nit, feel free to drop.

---

Updated this entire section to make it more relevant